en/tutorials/index.md

```diff
@@ -55,6 +55,40 @@ redirect_from: "/tutorials/"
         <p class="featured title">Todo Backend</p>
         <p style="color: rgb(85, 85, 85)">Learn how to build a Kitura backend for the Todo-Backend project.</p>
         <img class="featured-img" src="../../assets/TodoBackend.png" alt="ToDoBackend" width="90%">
-    </div></a>
+        </div></a>
 </div>
  
+
+<div class="books">
+    <p class="featured-header">Community Books &amp; Courses</p>
+    <div class="section-separator"></div>
+    
+    <a href="https://www.hackingwithswift.com/store/server-side-swift">
+    <div class="featured-item left" style="background-color: rgb(0, 0, 0); height: 345px; padding-bottom: 0px;">
+        <p class="featured title" style="font-family: 'Raleway'; margin: auto;">Server-Side Swift</p>
+        <p style="color: white; font-family: 'Raleway';">This course teaches you everything you need to craft your own server-side APIs and websites, all using Swift.</p>
+        <img class="featured-img" src="../../assets/Store%20Splash.png" alt="Hacking with Swift" width="100%" style="margin-bottom: 0;">
+    </div></a>
+    
+    <a href="https://nocturnalsolutions.gitbooks.io/kitura-book/content/">
+    <div class="featured-item right"  style="overflow: hidden; height: 345px;">
+        <p class="featured title" style="margin: auto;">Kitura Until Dawn</p>
+        <p style="color: rgb(85, 85, 85)">Detailed guide on using the Kitura framework, authored by Nocturnal, a member of the Swift community.</p>
+        <img class="featured-img" src="../../assets/kitura-until-dawn-screengrab.png" alt="Kitura Until Dawn" width="100%">
+    </div></a>
+    
+    <a href="https://www.udemy.com/courses/search/?q=Kitura&src=ukw">
+    <div class="featured-item left"  style="background-color: rgb(235, 84, 87); overflow: hidden; height: 345px;">
+        <p class="featured title" style="margin: auto; color: white;">Udemy Kitura Courses</p>
+        <p style="color: white">Enroll in high quality and engaging courses.</p>
+        <img class="featured-img" src="../../assets/kitura@2x.png" alt="Udemy" width="70%" style="padding-top: 30px;">
+    </div></a>
+    
+    <a href="https://swift-at-ibm.slack.com/messages/CAKNJL9K3">
+    <div class="featured-item right"  style="background-color: white; height: 345px; z-index: 1;">
+        <p class="featured title" style="margin: auto;">Submit Content</p>
+        <p style="color: rgb(85, 85, 85)">Feel your work belongs on kitura.io? Click here to Slack us a submission.</p>
+        <img class="featured-img" src="../../assets/plus.png" alt="Submit" width="40%" style="padding-top: 10px; opacity: 0.5; padding-bottom: 10px;">
+        <p style="z-index: 10; margin-top: 10px;"><a href="mailto:kye.maloy@ibm.com?Subject=Addition%20To%20Kitura.io">Don't have Slack? Send us an email.</a></p>
```

Comment:
I think this should just be `<a href="https://slack.kitura.io">slack.kitura.io</a>`